specification/containerservice/Fleet.Management/clusterresourceplacement.tsp

```diff
@@ -0,0 +1,229 @@
+import "@azure-tools/typespec-azure-resource-manager";
```

Comment:
Please can you make this file include only things that are strict subset of placement.kubernetes-fleet.io/v1 ?

Is it possible / hard / appropriate to make this a separate namespace?